snips/README.md

```diff
@@ -0,0 +1,104 @@
+# Hass.io Core Add-on: Snips.AI
+
+Support for Snips.ai voice assistant.
+
+![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] 
```

Comment:
Add all architectures to the list, use coloring (red/green) to indicate if it is supported or not.

Reply:
fixed in f9843167e364d49bcc634300ddd3aaa65bef48e0